pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -521,7 +509,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListeners(env *model.E
 				listenerOpts.filterChainOpts = buildSidecarOutboundTCPTLSFilterChainOpts(env, node, push, configs,
 					destinationIPAddress, service, servicePort, proxyLabels, meshGateway)
 			default:
-				// UDP or other protocols: no need to log, it's too noisy
+				log.Warnf("Unsupported inbound protocol for port %#v", servicePort)
```

Comment:
Too noisy. Please remove this. By default we treat everything as tcp (except for udp).
This line caused a lot of noise earlier. That’s the reason we removed it.

Reply:
OK, will revert it.